datastore/integration-test/response/metadata_collections_all_collections.json

```diff
@@ -189,73 +189,73 @@
                     "rodeo:standard_name": "air_pressure_at_sea_level",
                     "rodeo:level": 1.0
                 },
-                "air_temperature:0.1:minimum:PT10M": {
+                "air_temperature:0.0:minimum:PT12H": {
```

Comment:
Any idea why the order changed?

Reply:
Best guess is that it is now sorted and was not in the past. I don't think the diff cares about the order of keys.